Userland/Libraries/LibCrypto/ASN1/DER.h

```diff
@@ -209,28 +197,23 @@ class Decoder {
         return with_type_check<ValueType>(data);
     }
 
-    Result<Tag, DecodeError> read_tag();
-    Result<size_t, DecodeError> read_length();
-    Result<u8, DecodeError> read_byte();
-    Result<ReadonlyBytes, DecodeError> read_bytes(size_t length);
+    ErrorOr<Tag> read_tag();
+    ErrorOr<size_t> read_length();
+    ErrorOr<u8> read_byte();
+    ErrorOr<ReadonlyBytes> read_bytes(size_t length);
 
-    static Result<bool, DecodeError> decode_boolean(ReadonlyBytes);
-    static Result<UnsignedBigInteger, DecodeError> decode_arbitrary_sized_integer(ReadonlyBytes);
-    static Result<StringView, DecodeError> decode_octet_string(ReadonlyBytes);
-    static Result<nullptr_t, DecodeError> decode_null(ReadonlyBytes);
-    static Result<Vector<int>, DecodeError> decode_object_identifier(ReadonlyBytes);
-    static Result<StringView, DecodeError> decode_printable_string(ReadonlyBytes);
-    static Result<BitStringView, DecodeError> decode_bit_string(ReadonlyBytes);
+    static ErrorOr<bool> decode_boolean(ReadonlyBytes);
+    static ErrorOr<UnsignedBigInteger> decode_arbitrary_sized_integer(ReadonlyBytes);
+    static ErrorOr<StringView> decode_octet_string(ReadonlyBytes);
+    static ErrorOr<nullptr_t> decode_null(ReadonlyBytes);
+    static ErrorOr<Vector<int>> decode_object_identifier(ReadonlyBytes);
+    static ErrorOr<StringView> decode_printable_string(ReadonlyBytes);
+    static ErrorOr<BitStringView> decode_bit_string(ReadonlyBytes);
 
     Vector<ReadonlyBytes> m_stack;
     Optional<Tag> m_current_tag;
 };
 
-void pretty_print(Decoder&, DeprecatedOutputStream&, int indent = 0);
+ErrorOr<void> pretty_print(Decoder&, DeprecatedOutputStream&, int indent = 0);
```

Comment:
Also it seems that this method is not even used?

Reply:
That is, indeed, true.